packages/agents-core/src/mcp.ts

```diff
@@ -463,6 +464,7 @@ export interface MCPServerStreamableHttpOptions {
   reconnectionOptions?: any;
   sessionId?: string;
   // ----------------------------------------------------
+  timeout?: number;
```

Comment:
can you move this up above the OAuth related ones? More specifically, I would like to place this right after `toolFilter` in the code.